City-of-Helsinki/drupal-hdbt
# UHF-5 Base theme

How to test:
- Clone this repository and checkout this branch
- Type `nvm use && npm i && npx husky i` to setup the build tools + linters
- Type `npm run build` and check that the `dist` folder has compiled content (css/js/sprite icons)
- Change something from existing JS or SCSS files and run `git add . && git commit`
   - Check that the linters will run and check&fix your code
